config/routes.rb

```diff
@@ -119,6 +118,8 @@ def scoped_navigation_routes(context, navigation)
       resources :documents, only: [:index]
       resources :notes, only: [:index]
     end
+    resources :documents, only: [:edit, :update, :show]
```

Comment:
I think it would make sense to namespace the admin/case management routes similar to how we do the Zendesk stuff... I ran into a couple namespace issues by having them unscoped, and had to make the decision of whether to nest the documents routes under clients in order to "force" some sort of namespacing, but that added unnecessary complexity to the forms. For now, I moved the documents routes that are case-management related down near the case management routes so that at least there is a logical separation in the routes file. Namespacing also has some other benefits for code/route organization and access control. Hopefully we can talk about name spacing in the gardening meeting!